cmd/assessment/testdata/testGasSchedule.toml

```diff
@@ -0,0 +1,611 @@
+[BuiltInCost]
```

Comment:
this is a benchmark - you could have reused arwen.MakeGasSchedule() public function.

Reply:
no need for this toml file

Reply:
Right. Will use a test gas map as it doesn't matter what kind of gas map I use, I will simply get the same execution time results.

Reply:
done